### PR TITLE
Update calibre to 3.35.0

### PR DESCRIPTION
**Pull request by zcid**
- Added python-enum34 as a makedep. Required since PyQt 5.11.2

#5412

### CODE REVIEW

**Comment by CameronNemo** [comment]
Use the commit message "calibre: update to 3.35.0."

**Comment by zcid** [comment]
Sorry. I found the section on commit message formatting. Fixed.

File: srcpkgs/calibre/template [hunk 1]
```diff
@@ -21,7 +21,7 @@ maintainer="Orphaned <orphan@voidlinux.eu>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/calibre/template]: please use spdx identifier for the license

Comment (zcid) [reply srcpkgs/calibre/template]: Corrected the license to SPDX format

Comment (zcid) [reply srcpkgs/calibre/template]: Source on the license being GPL-3.0-only: https://www.mobileread.com/forums/showthread.php?s=673810cac0038765ef1402e695172891&t=313261

**Pull request merged**